From 1cf5b2400aefe61a7b24c189a1d7b3e219159a1f Mon Sep 17 00:00:00 2001
From: Cheick Traore <cheick.traore@foss.st.com>
Date: Thu, 10 Oct 2024 11:57:18 +0200
Subject: [PATCH] arm64: dts: st: Add autonomous clock for lpuart1 on stm32mp25
 SoC

Add autonomous mode clock for lpuart1 in stm32mp251.dtsi.

Change-Id: Icb3db348cc23523a15ed95fa5986ba184aadf6c1
Signed-off-by: Cheick Traore <cheick.traore@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/410083
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Cheick TRAORE <cheick.traore-ext@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Cheick TRAORE <cheick.traore-ext@st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2015,7 +2015,8 @@
 				compatible = "st,stm32h7-uart";
 				reg = <0x46030000 0x400>;
 				interrupts-extended = <&exti2 26 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&rcc CK_KER_LPUART1>;
+				clocks = <&rcc CK_KER_LPUART1>,
+					 <&rcc CK_LPUART1_AM>;
 				dmas = <&hpdma 166 0x20 0x10012>,
 				       <&hpdma 167 0x20 0x3021>;
 				dma-names = "rx", "tx";
